k8s/prod/common-overlay/probate/kustomization.yaml

```diff
@@ -10,4 +10,4 @@ patchesStrategicMerge:
 - ../../../namespaces/probate/probate-submit-service/prod.yaml
 - ../../../namespaces/probate/probate-frontend/prod.yaml
 - ../../../namespaces/probate/probate-orchestrator-service/prod.yaml
-- ../../../namespaces/probate/probate-back-office/prod.yaml
\ No newline at end of file
+- ../../../namespaces/probate/probate-back-office/prod.yaml
```

Comment:
Is this needed / relevant to this PR?

Reply:
This is the existing one, we have separate probate-base-office service running on prod